package.json

```diff
@@ -24,18 +28,27 @@
   "jest": {
     "verbose": true
   },
+  "engines": {
+    "node": ">=8.7.0"
+  },
   "dependencies": {
-    "virtualfs": "^2.2.0"
+    "lodash": "^4.5.0",
```

Comment:
I'd prefer we didn't use lodash. These are meant for applications, not libraries.

Reply:
lodash usage was inconsequential and I've replaced it with native methods